src/storefront/package.json

```diff
@@ -12,9 +12,9 @@
   },
   "dependencies": {
     "axios": "^0.21.1",
-    "core-js": "^3.0.1",
-    "echarts": "^4.2.1",
-    "uikit": "3.1.4"
+    "core-js": "^3.12.1",
+    "echarts": "^5.1.1",
+    "uikit": "3.6.21"
```

Comment:
Has this bump been tested e2e? `uikit` had been locked to 3.1.4 due to a prior breaking change. Haven't reviewed latest...

Reply:
I've tested manually with the test image and production image. Follow the whole process and didn't found any issues. Using Api Mock and Standard.

Reply:
Do you know what was breaking? Suggests any other tests?

Reply:
I seem to recall there was a problem with the switcher used in login/register
![image](https://user-images.githubusercontent.com/8783541/118288399-a36cb480-b491-11eb-8d5f-987cd6bb5d1e.png)

Reply:
@mvandervliet, This was using `mushop/mushop-storefront:2.2.0-RC0`. Does not see issues. Tested with local Kubernetes.

![mushop-storefront-RC0](https://user-images.githubusercontent.com/551007/118293233-0f094e80-b49f-11eb-9775-56f6d5054e4e.gif)

Currently the only security update that was not able to include on this PR, was on the API, for the `connect-redis` that has a "high risk vulnerability". Currently using v3.4.2. Updating to v4, redis is not pre-packaged, and need to change the code. I've changed everything, but shows connect error to the mushop-session, as appear to create incompatibility. I've rollbacked the change and will get back in the future. Will probably need to update mushop-session first.